apps/site/src/views/Schedule/components/CountdownItem/CountdownItem.tsx

```diff
@@ -0,0 +1,21 @@
+import { AnimatePresence, motion } from "framer-motion";
+import styles from "./CountdownItem.module.scss";
+
+interface CountdownItemProps {
+	num: number;
+	text: string;
+}
+
+export default function CountdownItem({ num, text }: CountdownItemProps) {
+	return (
+		<>
+			<div className={styles.countdownItem}>
+				<AnimatePresence mode="popLayout">
+					<motion.h2 key={num}>{num}</motion.h2>
```

Comment:
Issue: using heading elements for the countdown values obstructs the document outline. If you need text styles matching headings, Bootstrap provides utility classes, e.g `.h2` or `.fs-2` for just the size.